src/pages/AdminPage.tsx

```diff
@@ -1,8 +1,19 @@
+import { Box, Toolbar, Typography } from "@mui/material";
 import React from "react";
+import AdminTaskbar from "../components/AdminTaskbar";
 
 const AdminPage = () => {
 	return (
-		<div>AdminPage</div>
+		<div>
+			<AdminTaskbar />
+			<Box component="main" sx={{ flexGrow: 1, p: 15 }}>
+				<Toolbar />
+				<Typography paragraph align="center">
+          Admin Page
```

Comment:
**suggestion (non-blocking)**: Ditto formatting suggestion, sometimes the content has trailing tabs, sometimes not, sometimes is weirdly indented